### PR TITLE
[HTML5] - Actions Button Fix

### PR DESCRIPTION
**Pull request by KDSBrowne**
Makes the Actions button only render for Presenter.

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/actions-bar/service.js [hunk 1]
```diff
@@ -8,13 +9,9 @@ import Audio from '/imports/ui/components/audio-modal/component';
 let isUserPresenter = () => {
 
   // check if user is a presenter
-  let isPresenter = Users.findOne({
+  return isPresenter = Users.findOne({
```
Comment (Gcampes) [inline bigbluebutton-html5/imports/ui/components/actions-bar/service.js]: You can just return the `Users.findOne....` here. No point in having the `isPresenter = ...`

File: bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx [hunk 2]
```diff
@@ -29,34 +31,9 @@ export default class ActionsBar extends Component {
...
-        <div className={styles.center}>
-          <MuteAudioContainer />
-          <JoinAudioOptionsContainer
-            handleJoinAudio={this.props.handleOpenJoinAudio}
-            handleCloseAudio={this.props.handleExitAudio}
-
-          />
-          {/*<JoinVideo />*/}
-          <EmojiContainer />
+          <ActionsDropdown {...{isUserPresenter}}/>
```
Comment (Gcampes) [inline bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx]: Why is `<ActionsDropdown>` in 2 places in this code if there is only one showing in the application?

File: bigbluebutton-html5/imports/ui/components/actions-bar/service.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```
Comment (Gcampes) [inline bigbluebutton-html5/imports/ui/components/actions-bar/service.js]: Probably forgot to remove, I don't see it being used.

Comment (Gcampes) [inline bigbluebutton-html5/imports/ui/components/actions-bar/service.js]: Nevermind, this is needed.

File: bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx (hunk 2 above)
Comment (KDSBrowne) [inline bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx]: The second <ActionsDropdown> is there to assist in the positioning of the other buttons. Without it the rest of the buttons on the actions bar do not line up symmetrically with the rest of the page.

Comment (Gcampes) [inline bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx]: That doesn't seem like a good practice, you should fix the centralization issue with css only. What you could do to achieve the centralization you desire is: make the `center` div absolute and have it's width be the entire width of the actions bar.

**Pull request merged**